doc/user_guide/dynamo0p3.rst

```diff
@@ -112,13 +116,13 @@ Please see the :ref:`algorithm-layer` section for a description of the
 Objects in the LFRic API can be categorised by their functionality
 as data structures and information that specifies supported operations on
 a particular data structure. These data structures are represented by the
-five LFRic (Dynamo 0.3) API argument types: :ref:`scalar <lfric-scalar>`,
-:ref:`field <lfric-field>`, :ref:`field vector <lfric-field-vector>`,
-:ref:`operator <lfric-operator>` and :ref:`column-wise operator
-<lfric-cma-operator>`. All of them except the field vector are
-represented in the above example. ``qr`` represents a quadrature object
-which provides information required by a kernel to operate on fields
-(see section :ref:`dynamo0.3-quadrature` for more details).
+six LFRic (Dynamo 0.3) API argument types: :ref:`scalar <lfric-scalar>`,
+:ref:`array <lfric-array>`,:ref:`field <lfric-field>`, :ref:`field vector
```

Comment:
This isn't rendering correctly for me:
![image](https://github.com/stfc/PSyclone/assets/17409212/56d4f759-fc2d-4ece-b92c-f9777d25bcae)

Reply:
missing space issue, should be fixed